examples/simple/filter_flows.py

```diff
@@ -4,14 +4,15 @@
     mitmdump -s "flowfilter.py FILTER"
 """
 import sys
-from mitmproxy import flowfilter
+from mitmproxy import flowfilter, http
+import typing
 
 
 class Filter:
-    def __init__(self, spec):
+    def __init__(self, spec: str) -> None:
```

Comment:
`None` is required to pass mypy test. 
here is output without `None `
`filter_flows.py:12: error: The return type of "__init__" must be None`
I have cross checked with previous commits for extending mypy , and they all seems to using None with` __init__`